apps/web/components/Charts/MixedChart/MixedChart.tsx

```diff
@@ -28,60 +34,90 @@ export const MixedChart = ({ graphData }: GraphProps) => {
-        <YAxis yAxisId="left" stroke="#CCDFFF" tick={<CustomizedAxisTick />} />
-        <YAxis yAxisId="right" hide />
-        <Tooltip />
-        <Legend iconType="circle" align="right" content={RenderLegend} />
-        {parsedDatakeys.bars.map((item, index) => (
-          <Bar
-            key={index}
-            dataKey={item.datakey}
-            fill={item.color}
-            stackId={item.stackId}
-            barSize={16}
-            yAxisId="left"
-            radius={
-              index + 1 === parsedDatakeys.bars.length || !shouldStack
-                ? [20, 20, 0, 0]
-                : 0
-            }
-          />
-        ))}
-        {parsedDatakeys.yAxis?.left &&
-          parsedDatakeys.lines.map((item, index) => (
-            <Line
-              key={item.datakey}
-              dataKey={item.datakey}
-              stroke={item.color ? item.color : COLORS[index % COLORS.length]}
+        {parsedDatakeys.yAxis?.label && (
+          <Text variant="eyebrow">{parsedDatakeys.yAxis.label}</Text>
+        )}
+        {parsedDatakeys.yAxis?.labelRight && (
+          <Text variant="eyebrow">{parsedDatakeys.yAxis.labelRight}</Text>
+        )}
+      </Box>
+      <Box style={{ width: '100%', height: '90%' }}>
+        <ResponsiveContainer width="100%" height="100%">
+          <ComposedChart
+            width={250}
+            height={150}
+            data={parsedData}
+            margin={{
+              top: 30,
+              right: rightPadding,
+              left: 20,
+              bottom: 5,
+            }}
+          >
+            <CartesianGrid
+              strokeDasharray="0"
+              vertical={false}
+              stroke="#CCDFFF"
+            />
+            <XAxis
+              dataKey={parsedDatakeys.xAxis}
+              stroke="#CCDFFF"
+              tick={<CustomizedAxisTick />}
+              padding={{ left: 30 }}
+              tickLine={false}
+            />
+            <YAxis
+              yAxisId="left"
+              stroke="#CCDFFF"
+              tick={<CustomizedAxisTick />}
+            />
+            <YAxis
               yAxisId="right"
-              strokeWidth={3}
-              dot={{ r: 6, strokeWidth: 3 }}
+              orientation="right"
+              stroke="#CCDFFF"
+              tick={<CustomizedRightAxisTick />}
+              hide={!parsedDatakeys.yAxis?.showRight}
             />
-          ))}
-      </ComposedChart>
-    </ResponsiveContainer>
+            <Tooltip content={<CustomTooltip />} />
+            <Legend iconType="circle" align="right" content={RenderLegend} />
+            {parsedDatakeys.bars.map((item, index) => (
+              <Bar
+                key={index}
+                dataKey={item.datakey}
+                fill={item.color}
+                stackId={item.stackId}
+                barSize={16}
+                yAxisId="left"
+                radius={
+                  index + 1 === parsedDatakeys.bars.length || !shouldStack
```

Comment:
it's good practice to have the "subject" - ie. the thing that is being checked unchanged on one side of the equality operator.. makes it a little easier to understand

Reply:
so..